### DISCUSSION TITLE
Asynchronous Snapshot Testing

### DISCUSSION THREAD

**Discussion by skywalkerdude** [pointfreeco/swift-snapshot-testing#582] [Q&A]
I need some help doing asynchronous snapshot testing. I followed the [Gist here](https://github.com/pointfreeco/swift-snapshot-testing/blob/d48c197cd500f4471f7018745af3b074b990c4f3/Sources/SnapshotTesting/Extensions/Wait.swift), but it didn't seem to change anything. The details are here in this SO question: https://stackoverflow.com/questions/71360681/swiftui-snapshot-test-async-view, but basically, I am trying to find a way to test my custom `WrappedHStack` view that calculates dimensions on the fly, depending on how many items are in the view. Could someone give me pointers on how to get this working with swift snapshot testing?

I did a search and the closest thing I found was https://github.com/pointfreeco/swift-snapshot-testing/discussions/501, which seems like a different issue.

**Comment by dfeinzimer** [comment]
See also #701 and #669.
